### PR TITLE
Update CODEOWNER

### PR DESCRIPTION
**Pull request by apeforest**
## Description ##
Add myself to review modules

### CODE REVIEW

**Comment by marcoabreu** [comment]
Try rebase instead of merge :)

File: docs/tutorials/amp/amp_tutorial.md [hunk 1]
```diff
@@ -262,19 +262,20 @@ Below, we demonstrate for a gluon model and a symbolic model:
 ```python
 with mx.Context(mx.gpu(0)):
     # Below is an example of converting a gluon hybrid block to a mixed precision block
-    model = get_model("resnet50_v1")
-    model.collect_params().initialize(ctx=mx.current_context())
-    model.hybridize()
-    model(mx.nd.zeros((1, 3, 224, 224)))
-    converted_model = amp.convert_hybrid_block(model)
+    with warnings.catch_warnings(record=True) as w:
```
Comment (TaoLv) [inline docs/tutorials/amp/amp_tutorial.md]: Are these changes intended in this PR?

**Comment by karan6181** [comment]
@mxnet-label-bot add [pr-awaiting-review]
